modules/flowable-engine-common/src/main/java/org/flowable/common/engine/impl/el/function/VariableGetOrDefault2ExpressionFunction.java

```diff
@@ -0,0 +1,25 @@
+package org.flowable.common.engine.impl.el.function;
+
+import org.flowable.common.engine.api.variable.VariableContainer;
+import org.flowable.common.engine.impl.javax.el.PropertyNotFoundException;
+
+/**
+ * Returns the value of a variable (even if it's null), or a default if the variable is not found.
+ * This avoids the {@link PropertyNotFoundException} that otherwise gets thrown when referencing a variable in JUEL.
+ *
+ * @author Balavivek Bala Naga Sethuraja
+ */
+public class VariableGetOrDefault2ExpressionFunction extends AbstractFlowableVariableExpressionFunction {
+
+    public VariableGetOrDefault2ExpressionFunction() {
+        super("getOrDefault2");
+    }
+
+    public static Object getOrDefault2(VariableContainer variableContainer, String variableName, Object value) {
+        if(variableContainer.hasVariable(variableName)) {
```

Comment:
Project code formatting has a space between `if` and `(`.

Reply:
Project code formatting fixed.